docs/security.html

```diff
@@ -146,6 +145,7 @@
                 <li>ssl.enabled.protocols=TLSv1.2,TLSv1.1,TLSv1 (list out the SSL protocols that you are going to accept from clients. Do note that SSL is deprecated in favor of TLS and using SSL in production is not recommended)</li>
                 <li>ssl.keystore.type=JKS</li>
                 <li>ssl.truststore.type=JKS</li>
+                <li>ssl.truststore.password=test1234</li>
```

Comment:
Probably worth adding a comment stating that it's required for integrity checking.